### PR TITLE
[Debugger] Fix XML template evaluation support for latest runtime changes

### PR DESCRIPTION
**Pull request by NipunaRanasinghe**
## Purpose
Fixes https://github.com/ballerina-platform/ballerina-lang/issues/30483 - XML template evaluations produce XML parsing errors for XML sequence type.

## Check List 
- [x] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [ ] Updated Change Log
- [x] Checked Tooling Support (#<Issue Number>)
- [x] Added necessary tests
  - [ ] Unit Tests
  - [ ] Spec Conformance Tests
  - [x] Integration Tests
  - [ ] Ballerina By Example Tests
- [ ] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

### CODE REVIEW

File: misc/debug-adapter/modules/debug-adapter-core/src/main/java/org/ballerinalang/debugadapter/evaluation/engine/XMLTemplateEvaluator.java [hunk 1]
```diff
@@ -54,10 +55,9 @@ public BExpressionValue evaluate() throws EvaluationException {
             for (Node memberNode : syntaxNode.content()) {
                 xmlStrBuilder.append(memberNode.toSourceCode());
             }
-            Value xmlStrValue = VMUtils.make(context, xmlStrBuilder.toString()).getJdiValue();
-
-            RuntimeStaticMethod fromStringMethod = getRuntimeMethod(context, FROM_STRING_CLASS, XML_FROM_STRING_METHOD,
-                    Collections.singletonList(B_STRING_CLASS));
+            Value xmlStrValue = EvaluationUtils.getAsJString(context, xmlStrBuilder.toString());
+            RuntimeStaticMethod fromStringMethod = getRuntimeMethod(context, B_TYPE_CONVERTER_CLASS,
```
Comment (praveennadarajah) [inline misc/debug-adapter/modules/debug-adapter-core/src/main/java/org/ballerinalang/debugadapter/evaluation/engine/XMLTemplateEvaluator.java:59]: Shouldn't this variable name be `strToXmlMethod` after the changes?

File: misc/debug-adapter/modules/debug-adapter-core/src/main/java/org/ballerinalang/debugadapter/evaluation/utils/EvaluationUtils.java [hunk 2]
```diff
@@ -77,7 +78,7 @@
     public static final String B_TYPE_ARRAY_CLASS = RUNTIME_HELPER_PREFIX + "api.types.Type[]";
     private static final String B_LINK_CLASS = RUNTIME_HELPER_PREFIX + "api.values.BLink";
     private static final String B_ERROR_VALUE_CLASS = RUNTIME_HELPER_PREFIX + "internal.values.ErrorValue";
-    public static final String FROM_STRING_CLASS = "org.ballerinalang.langlib.xml.FromString";
+
```
Comment (praveennadarajah) [inline misc/debug-adapter/modules/debug-adapter-core/src/main/java/org/ballerinalang/debugadapter/evaluation/utils/EvaluationUtils.java:81]: Shall we remove this extra line?

File: misc/debug-adapter/modules/debug-adapter-core/src/main/java/org/ballerinalang/debugadapter/evaluation/engine/XMLTemplateEvaluator.java (hunk 1 above)
Comment (NipunaRanasinghe) [reply misc/debug-adapter/modules/debug-adapter-core/src/main/java/org/ballerinalang/debugadapter/evaluation/engine/XMLTemplateEvaluator.java]: Fixed in ae66fd25bdd34c10affa694460255a0f3bf4983f

File: misc/debug-adapter/modules/debug-adapter-core/src/main/java/org/ballerinalang/debugadapter/evaluation/utils/EvaluationUtils.java (hunk 2 above)
Comment (NipunaRanasinghe) [reply misc/debug-adapter/modules/debug-adapter-core/src/main/java/org/ballerinalang/debugadapter/evaluation/utils/EvaluationUtils.java:81]: Fixed in ae66fd25bdd34c10affa694460255a0f3bf4983f

**Review by praveennadarajah — APPROVED** [review]

**Pull request merged**